### PR TITLE
feat(mode): integrating button component

### PR DESCRIPTION
**Pull request by theJohnnyMe**
1. Adding button component - a couple of variations
2. Make a mode switcher remain in the same place so we can easily switch modes even when we are at the bottom of the page.

### CODE REVIEW

File: react/src/App.tsx [hunk 1]
```diff
@@ -5,85 +5,103 @@ import ModeSwitcher from "./components/ModeSwitcher";
...
+          </div>
+          <div className='sdds-u-w-100'>
+            <div className="sdds-headline-02 sdds-u-pb1">Buttons</div>
+            <sdds-button
+                type="primary"
+                size="lg"
+                fullbleed
+                text="Primary fullbleed button"
+            ></sdds-button>
+            <div className="sdds-u-mt2 sdds-u-flex" style={{justifyContent: "space-around"}}>
```
Comment (theJohnnyMe) [inline react/src/App.tsx:89]: What do you think of introducing flex specific utility classes? 
For justify-content etc...

**Comment by theJohnnyMe** [comment]
I run intention formating with "2 spaces" so it shows more changes then I really added in the code...

File: react/src/App.tsx (hunk 1 above)
Comment (mJarsater) [reply react/src/App.tsx:89]: Yep, could absolutely be something to look into. Would be nice to know how many are using the utility classes though?

**Review by mJarsater — APPROVED** [review]

File: react/src/App.tsx (hunk 1 above)
Comment (theJohnnyMe) [reply react/src/App.tsx:89]: I agree with that as well ;) We would need to expand a bit on them and also raise awareness of their existence. I find them super useful when making layouts.

**Pull request merged**